### PR TITLE
[WIP] Add Swagger descriptions

### LINKED ISSUE #3830: Swagger Doumentation Descriptions and Verification Missing
We currently store descriptions of our API objects for Swagger in `api/definitions` but we are missing many definitions. 

We also do not have any verification that these descriptions exist.

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Fixes #3830
Fixes #3830
- [ ] api.patch
- [ ] integer
- [ ] json.watchevent
- [ ] patch.object
- [ ] runtime.rawextension
- [ ] v1.awselasticblockstorevolumesource
- [ ] v1.binding
- [ ] v1.build  (https://github.com/stevekuznetsov/origin/commit/ad8ae3227d2c7e4f89a0b56c6cc67a4e97050a4f) (@csrwng PTAL)
- [ ] v1.buildconfiglist (https://github.com/stevekuznetsov/origin/commit/ad8ae3227d2c7e4f89a0b56c6cc67a4e97050a4f) (@csrwng PTAL)
- [ ] v1.buildconfigspec (https://github.com/stevekuznetsov/origin/commit/ad8ae3227d2c7e4f89a0b56c6cc67a4e97050a4f) (@csrwng PTAL)
- [ ] v1.buildconfigstatus (https://github.com/stevekuznetsov/origin/commit/ad8ae3227d2c7e4f89a0b56c6cc67a4e97050a4f) (@csrwng PTAL)
- [ ] v1.buildlist (https://github.com/stevekuznetsov/origin/commit/ad8ae3227d2c7e4f89a0b56c6cc67a4e97050a4f) (@csrwng PTAL)
- [ ] v1.buildlog (https://github.com/stevekuznetsov/origin/commit/ad8ae3227d2c7e4f89a0b56c6cc67a4e97050a4f) (@csrwng PTAL)
- [ ] v1.buildoutput (https://github.com/stevekuznetsov/origin/commit/ad8ae3227d2c7e4f89a0b56c6cc67a4e97050a4f) (@csrwng PTAL)
- [ ] v1.buildrequest (https://github.com/stevekuznetsov/origin/commit/ad8ae3227d2c7e4f89a0b56c6cc67a4e97050a4f) (@csrwng PTAL)
- [ ] v1.buildsource (https://github.com/stevekuznetsov/origin/commit/ad8ae3227d2c7e4f89a0b56c6cc67a4e97050a4f) (@csrwng PTAL)
- [X] v1.buildspec (https://github.com/stevekuznetsov/origin/commit/ad8ae3227d2c7e4f89a0b56c6cc67a4e97050a4f) (@csrwng PTAL)
- [ ] v1.buildstatus (https://github.com/stevekuznetsov/origin/commit/ad8ae3227d2c7e4f89a0b56c6cc67a4e97050a4f) (@csrwng PTAL)
- [ ] v1.buildstrategy (https://github.com/stevekuznetsov/origin/commit/ad8ae3227d2c7e4f89a0b56c6cc67a4e97050a4f) (@csrwng PTAL)
- [ ] v1.buildtriggerpolicy (https://github.com/stevekuznetsov/origin/commit/ad8ae3227d2c7e4f89a0b56c6cc67a4e97050a4f) (@csrwng PTAL)
- [ ] v1.capabilities
- [ ] v1.capability
- [ ] v1.cephfsvolumesource
- [ ] v1.clusternetwork
- [ ] v1.clusternetworklist
- [ ] v1.clusterpolicy (https://github.com/stevekuznetsov/origin/commit/571899384732ed4c9be32b25e242a27f3f12a037) (@deads2k PTAL)
- [ ] v1.clusterpolicybinding (https://github.com/stevekuznetsov/origin/commit/571899384732ed4c9be32b25e242a27f3f12a037) (@deads2k PTAL)
- [ ] v1.clusterpolicybindinglist (https://github.com/stevekuznetsov/origin/commit/571899384732ed4c9be32b25e242a27f3f12a037) (@deads2k PTAL)
- [ ] v1.clusterpolicylist (https://github.com/stevekuznetsov/origin/commit/571899384732ed4c9be32b25e242a27f3f12a037) (@deads2k PTAL)
- [ ] v1.clusterrole (https://github.com/stevekuznetsov/origin/commit/571899384732ed4c9be32b25e242a27f3f12a037) (@deads2k PTAL)
- [ ] v1.clusterrolebinding (https://github.com/stevekuznetsov/origin/commit/571899384732ed4c9be32b25e242a27f3f12a037) (@deads2k PTAL)
- [ ] v1.clusterrolebindinglist (https://github.com/stevekuznetsov/origin/commit/571899384732ed4c9be32b25e242a27f3f12a037) (@deads2k PTAL)
- [ ] v1.clusterrolelist (https://github.com/stevekuznetsov/origin/commit/571899384732ed4c9be32b25e242a27f3f12a037) (@deads2k PTAL)
- [ ] v1.componentcondition
- [ ] v1.componentstatus
- [ ] v1.componentstatuslist
- [ ] v1.container
- [ ] v1.containerport
- [ ] v1.containerstate
- [ ] v1.containerstaterunning
- [ ] v1.containerstateterminated
- [ ] v1.containerstatewaiting
- [ ] v1.containerstatus
- [ ] v1.custombuildstrategy (https://github.com/stevekuznetsov/origin/commit/ad8ae3227d2c7e4f89a0b56c6cc67a4e97050a4f) (@csrwng PTAL)
- [ ] v1.customdeploymentstrategyparams
- [ ] v1.deleteoptions
- [ ] v1.deploymentcause
- [ ] v1.deploymentcauseimagetrigger
- [ ] v1.deploymentconfiglist
- [ ] v1.deploymentconfigrollback
- [ ] v1.deploymentconfigrollbackspec
- [ ] v1.deploymentconfigspec
- [ ] v1.deploymentconfigstatus
- [ ] v1.deploymentdetails
- [ ] v1.deploymentstrategy
- [ ] v1.deploymenttriggerimagechangeparams
- [ ] v1.deploymenttriggerpolicy
- [ ] v1.dockerbuildstrategy (https://github.com/stevekuznetsov/origin/commit/ad8ae3227d2c7e4f89a0b56c6cc67a4e97050a4f) (@csrwng PTAL)
- [ ] v1.emptydirvolumesource
- [ ] v1.endpointaddress
- [ ] v1.endpointport
- [ ] v1.endpoints
- [ ] v1.endpointslist
- [ ] v1.endpointsubset
- [ ] v1.envvar
- [ ] v1.envvarsource
- [ ] v1.event
- [ ] v1.eventlist
- [ ] v1.eventsource
- [ ] v1.execaction
- [ ] v1.execnewpodhook
- [ ] v1.finalizername
- [ ] v1.gcepersistentdiskvolumesource
- [ ] v1.gitbuildsource (https://github.com/stevekuznetsov/origin/commit/ad8ae3227d2c7e4f89a0b56c6cc67a4e97050a4f) (@csrwng PTAL)
- [ ] v1.gitrepovolumesource
- [ ] v1.gitsourcerevision (https://github.com/stevekuznetsov/origin/commit/ad8ae3227d2c7e4f89a0b56c6cc67a4e97050a4f) (@csrwng PTAL)
- [ ] v1.glusterfsvolumesource
- [ ] v1.group
- [ ] v1.grouplist
- [ ] v1.handler
- [ ] v1.hostpathvolumesource
- [ ] v1.hostsubnet
- [ ] v1.hostsubnetlist
- [ ] v1.httpgetaction
- [ ] v1.identity
- [ ] v1.identitylist
- [ ] v1.image
- [ ] v1.imagechangetrigger (https://github.com/stevekuznetsov/origin/commit/ad8ae3227d2c7e4f89a0b56c6cc67a4e97050a4f) (@csrwng PTAL)
- [ ] v1.imagelist
- [ ] v1.imagestream
- [ ] v1.imagestreamimage
- [ ] v1.imagestreamlist
- [ ] v1.imagestreammapping
- [ ] v1.imagestreamspec
- [ ] v1.imagestreamstatus
- [ ] v1.imagestreamtag
- [ ] v1.iscsivolumesource
- [ ] v1.lifecycle
- [ ] v1.lifecyclehook
- [ ] v1.limitrange
- [ ] v1.limitrangeitem
- [ ] v1.limitrangelist
- [ ] v1.limitrangespec
- [ ] v1.listmeta
- [ ] v1.loadbalanceringress
- [ ] v1.loadbalancerstatus
- [ ] v1.localobjectreference
- [ ] v1.metadatafile
- [ ] v1.metadatavolumesource
- [ ] v1.namedclusterrole
- [ ] v1.namedclusterrolebinding
- [ ] v1.namedrole
- [ ] v1.namedrolebinding
- [ ] v1.namedtageventlist
- [ ] v1.namedtagreference
- [ ] v1.namespace
- [ ] v1.namespacelist
- [ ] v1.namespacespec
- [ ] v1.namespacestatus
- [ ] v1.nfsvolumesource
- [ ] v1.nodeaddress
- [ ] v1.nodecondition
- [ ] v1.nodelist
- [ ] v1.nodespec
- [ ] v1.nodestatus
- [ ] v1.nodesysteminfo
- [X] v1.oauthaccesstoken 
- [X] v1.oauthaccesstokenlist 
- [X] v1.oauthauthorizetoken
- [X] v1.oauthauthorizetokenlist 
- [X] v1.oauthclient 
- [X] v1.oauthclientauthorization 
- [X] v1.oauthclientauthorizationlist 
- [X] v1.oauthclientlist 
- [ ] v1.objectfieldselector
- [ ] v1.objectmeta
- [ ] v1.objectreference
- [ ] v1.parameter
- [ ] v1.persistentvolumeaccessmode
- [ ] v1.persistentvolumeclaimlist
- [ ] v1.persistentvolumeclaimspec
- [ ] v1.persistentvolumeclaimstatus
- [ ] v1.persistentvolumeclaimvolumesource
- [ ] v1.persistentvolumelist
- [ ] v1.persistentvolumespec
- [ ] v1.persistentvolumestatus
- [ ] v1.podcondition
- [ ] v1.podlist
- [ ] v1.podspec
- [ ] v1.podstatus
- [ ] v1.podtemplate
- [ ] v1.podtemplatelist
- [ ] v1.podtemplatespec
- [ ] v1.policy (https://github.com/stevekuznetsov/origin/commit/571899384732ed4c9be32b25e242a27f3f12a037) (@deads2k PTAL)
- [ ] v1.policybinding (https://github.com/stevekuznetsov/origin/commit/571899384732ed4c9be32b25e242a27f3f12a037) (@deads2k PTAL)
- [ ] v1.policybindinglist (https://github.com/stevekuznetsov/origin/commit/571899384732ed4c9be32b25e242a27f3f12a037) (@deads2k PTAL)
- [ ] v1.policylist (https://github.com/stevekuznetsov/origin/commit/571899384732ed4c9be32b25e242a27f3f12a037) (@deads2k PTAL)
- [ ] v1.policyrule (https://github.com/stevekuznetsov/origin/commit/571899384732ed4c9be32b25e242a27f3f12a037) (@deads2k PTAL)
- [ ] v1.probe
- [ ] v1.projectlist
- [ ] v1.projectrequest
- [ ] v1.projectspec
- [ ] v1.projectstatus
- [ ] v1.rbdvolumesource
- [ ] v1.recreatedeploymentstrategyparams
- [ ] v1.replicationcontroller
- [ ] v1.replicationcontrollerlist
- [ ] v1.replicationcontrollerspec
- [ ] v1.replicationcontrollerstatus
- [ ] v1.resourceaccessreview (https://github.com/stevekuznetsov/origin/commit/571899384732ed4c9be32b25e242a27f3f12a037) (@deads2k PTAL)
- [ ] v1.resourcequota
- [ ] v1.resourcequotalist
- [ ] v1.resourcequotaspec
- [ ] v1.resourcequotastatus
- [ ] v1.resourcerequirements
- [ ] v1.role (https://github.com/stevekuznetsov/origin/commit/571899384732ed4c9be32b25e242a27f3f12a037) (@deads2k PTAL)
- [ ] v1.rolebinding (https://github.com/stevekuznetsov/origin/commit/571899384732ed4c9be32b25e242a27f3f12a037) (@deads2k PTAL)
- [ ] v1.rolebindinglist (https://github.com/stevekuznetsov/origin/commit/571899384732ed4c9be32b25e242a27f3f12a037) (@deads2k PTAL)
- [ ] v1.rolelist (https://github.com/stevekuznetsov/origin/commit/571899384732ed4c9be32b25e242a27f3f12a037) (@deads2k PTAL)
- [ ] v1.rollingdeploymentstrategyparams
- [ ] v1.routespec
- [ ] v1.routestatus
- [ ] v1.runasuserstrategyoptions
- [ ] v1.secret
- [ ] v1.secretlist
- [ ] v1.secretvolumesource
- [ ] v1.securitycontext
- [ ] v1.securitycontextconstraints
- [ ] v1.securitycontextconstraintslist
- [ ] v1.selinuxcontextstrategyoptions
- [ ] v1.selinuxoptions
- [ ] v1.serviceaccount
- [ ] v1.serviceaccountlist
- [ ] v1.servicelist
- [ ] v1.serviceport
- [ ] v1.servicespec
- [ ] v1.servicestatus
- [ ] v1.sourcebuildstrategy (https://github.com/stevekuznetsov/origin/commit/ad8ae3227d2c7e4f89a0b56c6cc67a4e97050a4f) (@csrwng PTAL)
- [ ] v1.sourcecontroluser (https://github.com/stevekuznetsov/origin/commit/ad8ae3227d2c7e4f89a0b56c6cc67a4e97050a4f) (@csrwng PTAL)
- [ ] v1.sourcerevision (https://github.com/stevekuznetsov/origin/commit/ad8ae3227d2c7e4f89a0b56c6cc67a4e97050a4f) (@csrwng PTAL)
- [ ] v1.status
- [ ] v1.statuscause
- [ ] v1.statusdetails
- [ ] v1.subjectaccessreview (https://github.com/stevekuznetsov/origin/commit/571899384732ed4c9be32b25e242a27f3f12a037) (@deads2k PTAL)
- [ ] v1.tagevent
- [ ] v1.tcpsocketaction
- [ ] v1.template
- [ ] v1.templatelist
- [ ] v1.tlsconfig
- [ ] v1.useridentitymapping
- [ ] v1.userlist
- [ ] v1.volume
- [ ] v1.volumemount
- [ ] v1.webhooktrigger (https://github.com/stevekuznetsov/origin/commit/ad8ae3227d2c7e4f89a0b56c6cc67a4e97050a4f) (@csrwng PTAL)

### CODE REVIEW

**Comment by liggitt** [comment]
I thought Derek already added these and David wrote a test to verify they existed.

**Comment by liggitt** [comment]
oh, type descriptions, not field.

**Comment by stevekuznetsov** [comment]
@liggitt Yep, #3832 verifies that they exist, wanna help write some? :)

**Comment by stevekuznetsov** [comment]
@smarterclayton Before we make headway on this I need to know if I am interpreting your original statement about descriptions correctly - we need a description for every API object tracked by Swagger, right? Do we also need one for all Kubernetes objects? Some you added were Kubernetes resources.

**Comment by smarterclayton** [comment]
Let's do the descriptions for our objects first.  Upstream is working to make it so that they will have upstream descriptions for these.  Once they have that we'll adopt their story (move the descriptions into the code) - so having the descriptions on our side, we'll be able to move ours into the API types, and get theirs for free.

**Comment by smarterclayton** [comment]
See
https://github.com/GoogleCloudPlatform/kubernetes/pull/10933#issuecomment-123820804

On Wed, Jul 22, 2015 at 2:46 PM, Steve Kuznetsov <notifications@github.com>
wrote:

> @smarterclayton <https://github.com/smarterclayton> Before we make
> headway on this I need to know if I am interpreting your original statement
> about descriptions correctly - we need a description for every API object
> tracked by Swagger, right? Do we also need one for all Kubernetes objects?
> Some you added were Kubernetes resources.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/3843#issuecomment-123823619>.
>



-- 
Clayton Coleman | Lead Engineer, OpenShift

**Comment by stevekuznetsov** [comment]
@csrwng PTAL - added description for all `pkg/build` objects.

File: api/definitions/v1.build/description.adoc [hunk 1]
```diff
@@ -0,0 +1 @@
+A build is the process of transforming input parameters into a resulting object. Most often, the process is used to transform source code into a runnable image. The OpenShift build system provides extensible support for build strategies that are based on selectable types specified in the build API. There are three build strategies available: link:https://access.redhat.com/beta/documentation/en/openshift-enterprise-30-architecture/chapter-3-core-concepts#docker-build[Docker build], link:https://access.redhat.com/beta/documentation/en/openshift-enterprise-30-architecture/chapter-3-core-concepts#source-build[Source-to-Image (S2I) Build], and link:https://access.redhat.com/beta/documentation/en/openshift-enterprise-30-architecture/chapter-3-core-concepts#custom-build[Custom Build]. A Build encapsulates the inputs needed to produce a new deployable image as a <<v1.BuildSpec>>, as well as the status of the execution and a reference to the Pod which executed the build as a <<v1.BuildStatus>>.
```
Comment (csrwng) [inline api/definitions/v1.build/description.adoc]: > A build is the process of transforming input parameters into a resulting object. Most often, the process is used to transform source code into a runnable image.

I'd say a build is the process of transforming source... something like: 
A build is the process of transforming source code into a set of artifacts. Typically, the resulting artifact is a runnable image.

> The OpenShift build system provides extensible support for build strategies that are based on selectable types specified in the build API. There are three build strategies available: 

Seems too wordy. More like:
The OpenShift build system provides support for the following build strategies:

File: api/definitions/v1.buildconfigstatus/description.adoc [hunk 2]
```diff
@@ -0,0 +1 @@
+A BuildConfigStatus contains current state of the <<v1.BuildConfig>> object. The status is stored as an integer andstores the number of the last triggered build.
```
Comment (csrwng) [inline api/definitions/v1.buildconfigstatus/description.adoc]: contains *the* current state...

> The status is stored as an integer andstores the number of the last triggered build.

The status contains the number of the last triggered build.

File: api/definitions/v1.buildlog/description.adoc [hunk 3]
```diff
@@ -0,0 +1 @@
+A BuildLog is the (unused) resource associated with the build log redirector for retrieving build logs from Kubernetes. This is not an API object anyone is expected to use and only serves an internal purpose.
```
Comment (csrwng) [inline api/definitions/v1.buildlog/description.adoc]: > A BuildLog is the (unused) resource associated with the build log redirector for retrieving build logs from Kubernetes.  

A BuildLog is used to document the output of the REST call to retrieve the logs of a build. It is not a concrete type and is used only for documentation.

File: api/definitions/v1.buildoutput/description.adoc [hunk 4]
```diff
@@ -0,0 +1 @@
+A BuildOutput is input to a build strategy and describes the Docker image that the strategy should produce. The BuildOutput defines an optional location to push the output of this build to. This value will be used to look up a Docker image repository to push to. The output types supported are <<v1.ImageStreamTag>> and `DockerImage`. In the case of an `ImageStreamTag`. the `ImageStreamTag` will be looked for in the namespace of the build unless another namespace is specified. The BuildOutput contains a <<v1.PushSecret>>that is the name of the <<v1.Secret>> that should be used for setting up the authentication for executing the Docker push to an authentication-enabled Docker Registry (or Docker Hub).
```
Comment (csrwng) [inline api/definitions/v1.buildoutput/description.adoc]: > A BuildOutput is input to a build strategy and describes the Docker image that the strategy should produce. The BuildOutput defines an optional location to push the output of this build to. This value will be used to look up a Docker image repository to push to. The output types supported ... 

A BuildOutput (optional) specifies the output location of a build. It contains a reference that can be resolved to a  Docker repository where the resulting build image should be pushed. The supported output reference types ... 

> The output types supported are <<v1.ImageStreamTag>> and `DockerImage`. 

The output types supported are <<v1.ImageStreamTag>> and `DockerImage`. If `DockerImage` is specified, the Name of the reference should contain a docker repository spec.

File: api/definitions/v1.buildrequest/description.adoc [hunk 5]
```diff
@@ -0,0 +1,2 @@
+A BuildRequest is the resource used to pass parameters to a build generator signalling that a new <<v1.Build>> is requested. The BuildRequest contains the (optional) revision information from the source repository used in the build as a <<v1.SourceRevision>>.
```
Comment (csrwng) [inline api/definitions/v1.buildrequest/description.adoc]: > the resource used to pass parameters to a build generator

the resource used to pass parameters to *the* build generator endpoint,

File: api/definitions/v1.buildsource/description.adoc [hunk 6]
```diff
@@ -0,0 +1 @@
+A BuildSource is an object describing the SCM used for the <<v1.Build>>. A BuildSource contains a type string which defines the type of SCM used for the build. Currently, the only SCM supported by OpenShift is Git. As such, the BuildSource contains a <<v1.GitBuildSource>> to hold optional additional information about the Git buid source. The BuildSource also contains a string that defines the sub-directory in the SCM repository where the source code for the application exists. This allows to have buildable sources in directory other than root of a repository. The BuildSource also contains the source `Secret` that should be used for setting up the authentication for cloning private repository. The secret contains valid credentials for remote repository, where the data's key represent the authentication method to be used and value is the base64 encoded credentials. Supported auth methods are: ssh-privatekey.
```
Comment (csrwng) [inline api/definitions/v1.buildsource/description.adoc]: > ...to hold optional additional information about the Git buid source

to hold optional additional information about the Git *build* source

> The BuildSource also contains a string that defines the sub-directory in the SCM repository where the source code for the application exists. This allows to have buildable sources in directory other than root of a repository. 

The contextDir in the BuildSource can specify a sub-directory in the SCM repository to use for the build.

> for setting up the authentication for cloning private repository

for setting up the authentication for cloning *a* private repository

> The secret contains valid credentials for remote repository, where the data's key represent the authentication method to be used and value is the base64 encoded credentials. Supported auth methods are: ssh-privatekey.

The secret contains valid credentials for *the* remote source repository. The secret must contain a data key with the name of the authentication method to use and the credentials in base64 encoded form in the value. Currently, OpenShift only supports `ssh-privatekey` as an authentication method.

**Comment by stevekuznetsov** [comment]
@csrwng comments addressed

**Comment by stevekuznetsov** [comment]
@pmorie  [Travis panic](https://travis-ci.org/openshift/origin/jobs/72456057#L1559) on `TestPluginRootContextSet-2:
```
--- FAIL: TestPluginRootContextSet-2 (0.00s)
	empty_dir_test.go:253: Expected 1 chcon calls, got 0
panic: runtime error: index out of range [recovered]
	panic: runtime error: index out of range
```

**Comment by smarterclayton** [comment]
In the merge queue

On Fri, Jul 24, 2015 at 12:10 PM, Steve Kuznetsov <notifications@github.com>
wrote:

> @pmorie <https://github.com/pmorie> Travis panic
> <https://travis-ci.org/openshift/origin/jobs/72456057#L1559> on
> `TestPluginRootContextSet-2:
>
> --- FAIL: TestPluginRootContextSet-2 (0.00s)
>     empty_dir_test.go:253: Expected 1 chcon calls, got 0
> panic: runtime error: index out of range [recovered]
>     panic: runtime error: index out of range
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/3843#issuecomment-124569421>.
>



-- 
Clayton Coleman | Lead Engineer, OpenShift

File: api/definitions/v1.build/description.adoc [hunk 7]
```diff
@@ -0,0 +1 @@
+A build is the process of transforming source code into a resulting set of artifacts. Typically, the resulting artifacts are runnable Docker images. Most often, the process is used to transform source code into a runnable image. The OpenShift build system provides support for three build strategies: link:https://access.redhat.com/beta/documentation/en/openshift-enterprise-30-architecture/chapter-3-core-concepts#docker-build[Docker build], link:https://access.redhat.com/beta/documentation/en/openshift-enterprise-30-architecture/chapter-3-core-concepts#source-build[Source-to-Image (S2I) Build], and link:https://access.redhat.com/beta/documentation/en/openshift-enterprise-30-architecture/chapter-3-core-concepts#custom-build[Custom Build]. A Build encapsulates the inputs needed to produce a new deployable image as a <<v1.BuildSpec>>, as well as the status of the execution and a reference to the Pod which executed the build as a <<v1.BuildStatus>>.
```
Comment (csrwng) [inline api/definitions/v1.build/description.adoc]: > Most often, the process is used to transform source code into a runnable image.

pls remove

**Comment by danmcp** [comment]
@stevekuznetsov Bump and needs a rebase.

**Comment by stevekuznetsov** [comment]
@csrwng PTAL at `buildapi` for last-pass

**Comment by stevekuznetsov** [comment]
@deads2k PTAL authorization first-pass

**Comment by stevekuznetsov** [comment]
@liggitt  PTAL `oauthapi` first-pass

**Comment by liggitt** [comment]
> @liggitt PTAL oauthapi first-pass

looks fine

**Comment by danmcp** [comment]
@stevekuznetsov Bump

**Comment by stevekuznetsov** [comment]
@danmcp I'll break this into manageable chunks this week.

**Comment by smarterclayton** [comment]
With the rebase coming in our model here is changing a bit.  May need to
wait.

On Wed, Sep 16, 2015 at 12:21 AM, Steve Kuznetsov <notifications@github.com>
wrote:

> @danmcp <https://github.com/danmcp> I'll break this into manageable
> chunks this week.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/3843#issuecomment-140622955>.
>



-- 
Clayton Coleman | Lead Engineer, OpenShift

**Comment by stevekuznetsov** [comment]
@smarterclayton Thanks for the heads up, I'll hold for now

**Comment by deads2k** [comment]
> With the rebase coming in our model here is changing a bit.  May need to
> wait.

@smarterclayton can you elaborate on what's changing?  I know that the swagger might actually be used by code, but I'm not clear why that would affect our descriptions.

**Comment by smarterclayton** [comment]
The descriptions for objects now come from GoDoc on the types and are
generated into a map.

On Wed, Sep 16, 2015 at 9:01 AM, David Eads <notifications@github.com>
wrote:

> With the rebase coming in our model here is changing a bit. May need to
> wait.
>
> @smarterclayton <https://github.com/smarterclayton> can you elaborate on
> what's changing? I know that the swagger might actually be used by code,
> but I'm not clear why that would affect our descriptions.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/3843#issuecomment-140734910>.
>



-- 
Clayton Coleman | Lead Engineer, OpenShift

**Comment by stevekuznetsov** [comment]
Closing this, will approach using the new tools.
